src/app/home/page.tsx

```diff
@@ -0,0 +1,21 @@
+'use client'
+import HeroBanner from '@/components/HeroBanner';
+import Navbar from '@/components/Navbar';
+import LoginModal from '@/components/modals/LoginModal';
+import { useState } from 'react';
+
+const Home = () => {
+	const [closeModal, setCloseModal] = useState(true)
+	return (
+		<div className="min-h-full w-full flex">
+			<div className="w-2/12">{/***sidebar */}</div>
+			<div className="w-10/12">
+				<Navbar  closeModal={setCloseModal} />
+				<HeroBanner />
+			</div>
+			{!closeModal && <LoginModal closeModal={setCloseModal}/>}
+		</div>
+	);
+};
+
+export default Home;
```

Comment:
This component is duplicated, you've implemented it into many PRs, to make it easy ... you can just create  it own PR that  can be quickly merged

Reply:
Which component are you talking about exactly ?

Reply:
I meant this selected component was implemented in an other PR again